youtube_dl/extractor/vvvvid.py

```diff
@@ -156,3 +163,56 @@ def f(m):
             'view_count': int_or_none(video_data.get('views')),
             'like_count': int_or_none(video_data.get('video_likes')),
         }
+
+
+class VVVVIDShowIE(VVVVIDIE):
+    _VALID_URL = r'https?://(?:www\.)?vvvvid\.it/(?:#!)?(?:show)/(?P<show_id>\d+)/(?P<id>[^/]+)(?:\?|\&|/$|$)'
+    _TESTS = [{
+        'url': 'https://www.vvvvid.it/show/156/psyco-pass',
+        'info_dict': {
+            'id': '156',
+            'title': 'Psycho-Pass',
+        },
+        'playlist_count': 46,
+    }]
+
+    def _real_extract(self, url):
+        show_id, video_id = re.match(self._VALID_URL, url).groups()
+
+        response = self._download_info(
+            'https://www.vvvvid.it/vvvvid/ondemand/%s/seasons/' % show_id, video_id)
+
+        show_infos = self._download_info(
+            'https://www.vvvvid.it/vvvvid/ondemand/%s/info/' % show_id, video_id)
```

Comment:
should not be fatal, the `playlist_title` and `playlist_description` are not mandatory.